packages/okta-auth-js/lib/browser/browserStorage.js

```diff
@@ -76,13 +76,40 @@ storageUtil.getCookieStorage = function(options) {
     throw new AuthSdkError('getCookieStorage: "secure" and "sameSite" options must be provided');
   }
   return {
-    getItem: storageUtil.storage.get,
-    setItem: function(key, value) {
-      // Cookie shouldn't expire
-      storageUtil.storage.set(key, value, '2200-01-01T00:00:00.000Z', {
-        secure: secure, 
-        sameSite: sameSite
-      });
+    _getKey: function(key, subKey) {
+      return subKey ? `${key}_${subKey}` : key;
+    },
+    getItem: function(key, useAsPrefix /* optional */) {
+      // Get all data under key scope (prefixed with key)
+      if (useAsPrefix) {
+        var data = storageUtil.storage.get();
+        var value = Object.keys(data).reduce((acc, curr) => {
```

Comment:
`reduce` is not supported for IE11. Can we use `forEach` instead?

Reply:
How about a polyfill? or it's too heavy for this lib?

Reply:
Changed to `forEach`